File: app/src/main/java/com/alvarotobita/coletadados/storage/AppDatabase.java
package com.alvarotobita.coletadados.storage;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import com.alvarotobita.coletadados.model.DataSample;

@Database(entities = {DataSample.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract DataSampleDao dataSampleDao();
}

File: app/src/main/java/com/example/v3mvp/data/AppDatabase.kt
package com.example.v3mvp.data

import android.content.Context
import androidx.room.Database
import androidx.room.Room
import androidx.room.RoomDatabase
import com.example.v3mvp.model.Coleta

@Database(entities = [Coleta::class], version = 5)
abstract class AppDatabase : RoomDatabase() {
    abstract fun coletaDao(): ColetaDao

    companion object {
        @Volatile private var INSTANCE: AppDatabase? = null

        fun getInstance(context: Context): AppDatabase {
            return INSTANCE ?: synchronized(this) {
                Room.databaseBuilder(
                    context.applicationContext,
                    AppDatabase::class.java,
                    "coleta.db"
                )
                    .fallbackToDestructiveMigration()
                    .build().also { INSTANCE = it }
            }
        }
    }
}
